src/coverlet.msbuild.tasks/Services.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace coverlet.msbuild.tasks
+{
+    internal static class Services
```

Comment:
Could be more elegant create a base class
```cs
public abstract class BaseTask : Task
{
   protected static IServiceProvider ServiceProvider { get;set; }
}
...
public class InstrumentationTask : BaseTask 
...
public class CoverageResultTask : BaseTask 
...
```
And build container 
```cs
...
ServiceProvider = serviceCollection.BuildServiceProvider();
...
```

Reply:
Done